src/projection.ts

```diff
@@ -4,11 +4,14 @@ import type { AtlasUser } from './user.js';
 import { AtlasDataset } from './project.js';
 import type { AtlasIndex } from './index.js';
 
+export type ProjectGetInfo = Record<string, any>;
```

Comment:
Ok for now but should get the OpenAPI types in soon.

Reply:
Yeah this is in the next PR in this series.